lynx/theme.json

```diff
@@ -195,45 +198,51 @@
 			}
 		},
 		"color": {
-			"background": "#fff",
+			"background": "var(--wp--preset--color--background)",
 			"text": "var(--wp--preset--color--foreground)"
 		},
 		"elements": {
 			"h1": {
 				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--system-font)",
```

Comment:
```suggestion
```

Small nitpick, but as you're already setting the same default font family at `styles.typography.fontFamily`, you shouldn't need to set it again for each heading element.

Reply:
Yes, you're absolutely right! Fixed!